src/components/Icon.js

```diff
@@ -15,19 +15,17 @@ import { colors, normalize } from '../config';
 export const Icon = ({ xml, width, height, name, size, focused, iconColor, style, iconStyle }) => {
   const color = iconColor || (focused ? colors.accent : colors.primary);
 
-  const hitSlopInicons = size || normalize(44) - size / 2;
-  const hitSlopSvg = width || normalize(44) - width / 2;
+  const hitSlop = (normalize(44) - size ?? width) / 2;
 
-  //to check the hitSlop values
-  console.warn(hitSlopInicons);
-  console.warn(hitSlopSvg);
+  //for testing
+  console.warn(xml ? 'hitslopSvg:' : 'hitslopFont:', hitSlop);
```

Comment:
this is the output: 
<img width="922" alt="Screenshot 2021-06-21 at 17 53 46" src="https://user-images.githubusercontent.com/43641321/123937402-69e6fe80-d996-11eb-8c70-8b2fabefaf33.png">

Reply:
remove this before merging

Reply:
done in f90ab7f